src/main/java/io/rubuy74/rhs/converter/EventDTOJSONConverter.java

```diff
@@ -0,0 +1,36 @@
+package io.rubuy74.rhs.converter;
+
+import io.rubuy74.rhs.dto.EventDTO;
+import io.rubuy74.rhs.utils.ValidatorUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Map;
+
+public class EventDTOJSONConverter {
+    public static Logger logger = LoggerFactory.getLogger(EventDTOJSONConverter.class);
+    public static EventDTO fromJson(Map<String,Object> rawPayload) {
+        if (rawPayload == null) {
+            logger.error("operation=deserialize_event_dto, " +
+                    "msg=Raw payload is null");
+            return null;
+        }
```

Comment:
it is a good idea to keep the "inner layers" of the architecture free of infrastructure details like logging and calls to framework methods. Usually we define a method like:

```java
public static void checkArgument(boolean expression, String errorMessage) {

  if (!expression) {
  throw new IllegalArgumentException(errorMessage);
  }
}
```

and then use it to throw errors on assertions. the exceptions are than handled (and logged) in the above layers

```suggestion
     checkArgument(rawPayload == null, "Raw payload cannot be null");
```